doc/develop/test/twister.rst

```diff
@@ -988,6 +988,17 @@ The ``--device-flash-with-test`` option indicates that on the platform
 the flash operation also executes a test scenario, so the flash timeout is
 increased by a test scenario timeout.
 
+Special hardware preliminary
```

Comment:
"Special hardware preliminary" is this a correct English term? Does it mean "Additional hardware requirements"

Reply:
The [`Fixtures`](https://github.com/zephyrproject-rtos/zephyr/blob/9d9089edd09919c90c4224222fc2c560410e6c85/doc/develop/test/twister.rst?plain=1#L1178) section below has some details already, so it looks better to add a link to the new page there.